javascript/javascript1/week1/House_Price.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Calculate and Evaluate House Price</title>
+        <meta charset="utf-8" name="viewport">
+    </head>
+    <body>
+
+        <script>
+
+
+            function evaluateHousePrice (name, width, height, depth, gardenSize, houseCost) 
+            {
+
+
+                volumeInMeters = width*height*depth;
```

Comment:
you need to write const or let before a variable to have function scope, now volumemeters automatically becomes a global variable.
you can read more about it 
https://www.programiz.com/javascript/variable-scope